build.gradle

```diff
@@ -47,6 +48,14 @@ pitest {
     threads = 4
 }
 
+gitHooks {
+    hooksDirectory = file('.git/hooks')
+    gradleCommand = './gradlew'
+    hooks = [
+        'pre-push': 'clean check test'
```

Comment:
Is it necessary to perform `test` after `check`? I believe `check` already includes `test`.

https://stackoverflow.com/questions/50104666/gradle-difference-between-test-and-check

Reply:
My bad, removed `test`.